frontend/packages/topology/src/actions/modify-application.ts

```diff
@@ -10,7 +10,7 @@ export const getModifyApplicationAction = (kind: K8sKind, obj: K8sResourceKind):
   return {
     id: 'modify-application',
     label: i18next.t('topology~Edit Application grouping'),
-    insertBefore: 'edit-pod-count',
+    insertBefore: ['edit-pod-count', 'set-traffic-distribution'],
```

Comment:
do we need this here as this has reference of knative stuff?

Reply:
without this, the Edit Application grouping would appear as the last option in the menu for knative services. Couldn't think of any other alternative cc @sahil143 @rohitkrai03

Reply:
@debsmita you can pass the insertBefore through function argument.